HOMEPAGE_README.md

```diff
@@ -0,0 +1,22 @@
+# REDI Sprint 2020 Frontend Development 2, Final Project
```

Comment:
Please make this README.md as it is the your final project concept.

Reply:
I didn't know how to rename it, so instead i updated the README.md that you created originally (and there is a pr for that).